### DISCUSSION TITLE
View Returning File / Download File

### DISCUSSION THREAD

**Discussion by pramos84** [emmett-framework/emmett#489] [Q&A] [answered]
Hi, guys. I'm trying to make a web interface to process excel files using pandas, doing some transformation in this files. However I can't return the final file to users. I think it's possible to copy the file to "static" folder and returns a link to users, but what I'm looking for is something like "send_file" function from Flask. Any tips on how to do this? Thanks in advance.

**Comment by gi0baro** [comment, marked as answer]
@pramos84 you can use the `stream_file` from `emmett.helpers` (https://github.com/emmett-framework/emmett/blob/master/emmett/helpers.py#L36). Please open an issue to add this into the documentation.

**Reply by pramos84** [reply]
Thanks, @gi0baro! I had test this function and it fill my needs. Would be nice if we could set the name of output file (default name is the last part of url + extension). Anyway, I'll open an issue to document this helper. Thanks again.
